multibody/kinematic/distance_evaluator.cc

```diff
@@ -78,37 +90,38 @@ VectorX<T> DistanceEvaluator<T>::EvalFullJacobianDotTimesV(
   auto pt_A_cast = pt_A_.template cast<T>();
   auto pt_B_cast = pt_B_.template cast<T>();
 
-
   // Perform all kinematic calculations, finding A, B in world frame,
   // Jacobians J_A and J_B, and Jdotv for both A and B
-  plant().CalcPointsPositions(context, frame_A_,
-      pt_A_cast, world, &pt_A_world);
-  plant().CalcPointsPositions(context, frame_B_,
-      pt_B_cast, world, &pt_B_world);
+  plant().CalcPointsPositions(context, frame_A_, pt_A_cast, world, &pt_A_world);
+  plant().CalcPointsPositions(context, frame_B_, pt_B_cast, world, &pt_B_world);
   auto rel_pos = pt_A_world - pt_B_world;
 
   MatrixX<T> J_A(3, plant().num_velocities());
   MatrixX<T> J_B(3, plant().num_velocities());
 
   plant().CalcJacobianTranslationalVelocity(
-    context, drake::multibody::JacobianWrtVariable::kV,
-    frame_A_, pt_A_cast, world, world, &J_A);
+      context, drake::multibody::JacobianWrtVariable::kV, frame_A_, pt_A_cast,
+      world, world, &J_A);
   plant().CalcJacobianTranslationalVelocity(
-    context, drake::multibody::JacobianWrtVariable::kV,
-    frame_B_, pt_B_cast, world, world, &J_B);
+      context, drake::multibody::JacobianWrtVariable::kV, frame_B_, pt_B_cast,
+      world, world, &J_B);
   auto J_rel = J_A - J_B;
 
-  VectorX<T> J_A_dot_times_v = plant().CalcBiasSpatialAcceleration(
-      context, drake::multibody::JacobianWrtVariable::kV, frame_A_,
-      pt_A_cast, world, world).translational();
-  VectorX<T> J_B_dot_times_v = plant().CalcBiasSpatialAcceleration(
-      context, drake::multibody::JacobianWrtVariable::kV, frame_B_,
-      pt_B_cast, world, world).translational();
+  VectorX<T> J_A_dot_times_v =
+      plant()
+          .CalcBiasSpatialAcceleration(
+              context, drake::multibody::JacobianWrtVariable::kV, frame_A_,
+              pt_A_cast, world, world)
+          .translational();
```

Comment:
I think we should be able to use `CalcBiasTranslationalAcceleration()` in place of `CalcBiasSpatialAcceleration().translational()`

Reply:
Good to know! I missed that one. Fixed in WorldPointEvaluator as well.